pre_commit/languages/golang.py

```diff
@@ -75,6 +75,7 @@ def get_env_patch(venv: str, version: str) -> PatchesT:
 
     return (
         ('GOROOT', os.path.join(venv, '.go')),
+        ('GOTOOLCHAIN', 'local'),
```

Comment:
does this break with `language_version: system`? this seems different than the install code below

Reply:
If the `language_version` is `system` the function is going to return before reaching that line, so it's consistent with the install code.